cosmo_tester/test_suites/bootstrap_based_tests/distributed_installation_test.py

```diff
@@ -0,0 +1,152 @@
+########
+# Copyright (c) 2018 Cloudify.co Ltd. All rights reserved
```

Comment:
we usually use `Cloudify Platform Ltd`, no? @idob-cloudify which one should it be?

Reply:
Fixed